vars/checkoutScm.groovy

```diff
@@ -0,0 +1,32 @@
+// vars/checkoutScm.groovy
+
+import com.intel.checkoutScmInternal
+
+/**
+ * checkoutSCM.groovy
```

Comment:
Is it supposed to be *checkoutScm* or *checkoutSCM*?  The file name is making actually be *checkoutScm*, but the comment above says otherwise.  Your [example usage and "test run"](https://github.com/daos-stack/scons_local/pull/1) of this patch seems to indicate the former also.

Additionally, [checkPatch](https://github.com/daos-stack/pipeline-lib/pull/8/files#diff-431fc0f8a5e4d1222eebedad5d5ef6ceR31) [above](https://github.com/daos-stack/pipeline-lib/pull/8/files#diff-431fc0f8a5e4d1222eebedad5d5ef6ceR47) is also referring to *checkoutSCM*.  This makes me wonder how [the test run using this PR](http://jenkins-3.wolf.hpdd.intel.com:8080/job/daos-stack-org/job/scons_local/job/PR-1/3) was successful.  So looking at it, it doesn't say which _pipeline-lib_ revision it's using because it's a restart of the [previous run](http://jenkins-3.wolf.hpdd.intel.com:8080/job/daos-stack-org/job/scons_local/job/PR-1/2) at _stage Phase 1_.

So looking at that [previous run](http://jenkins-3.wolf.hpdd.intel.com:8080/job/daos-stack-org/job/scons_local/job/PR-1/2), the pipeline-lib checkout used there was:

```
Loading library pipeline-lib@jemalmbe_corci-504
Examining daos-stack/pipeline-lib
Attempting to resolve jemalmbe_corci-504 as a branch
Resolved jemalmbe_corci-504 as branch jemalmbe_corci-504 at revision 0de9650495a7cd46bb8eca75ba564f244e57d2c7
```
which is not the sha1 of this revision of this PR, which is c1cc9b65e5f29a068706de539b8c0676bbda07e1.

So ultimately I think the test run for this PR needs to be redone so that it's done on this revision (c1cc9b65e5f29a068706de539b8c0676bbda07e1).  It should reveal this naming confusion issue.
```suggestion
 * checkoutScm.groovy
```

Reply:
Fixed.